modules/social_features/social_group/src/Plugin/views/field/SocialGroupViewsBulkOperationsBulkForm.php

```diff
@@ -197,7 +197,7 @@ public function viewsFormValidate(&$form, FormStateInterface $form_state) {
       // Grab all the actions that are available.
       foreach (Element::children($this->actions) as $action) {
         // If the option is not in our selected options, next.
-        if (($action_key = array_search($action, array_column($this->options['selected_actions'], 'action_id'))) === FALSE) {
+        if (($action_key = array_search($action, array_combine(array_keys($this->options['selected_actions']), array_column($this->options['selected_actions'], 'action_id')))) === FALSE) {
```

Comment:
Hi @hitesh-jain thank you for your contribution.

Can you please split the if statement to be more legible? 

Also when using `array_combine` there might be a case where `FALSE` is returned ( if for some reason those arrays have a different size), if that happens, `$action_key` value will be `NULL` and the code will continue, breaking on the same `$label->render()` point. We need to add more defensive checks to make this won't happen.

Kind regards.

Reply:
thanks @tbsiqueira for review. Let me check this today.